rust/noosphere-core/src/data/address.rs

```diff
@@ -99,15 +153,60 @@ impl LinkRecord {
                 }
             }
         }
-
         warn!("No facts contained a link!");
-
         None
     }
+
+    /// Return a base64-encoded string serialization of the
+    /// underlying [Ucan].
+    pub fn encode(&self) -> Result<String> {
+        self.0.encode()
+    }
+
+    /// Return a base64-encoded [Vec<u8>] of the underlying [Ucan].
+    pub fn as_bytes(&self) -> Result<Vec<u8>> {
```

Comment:
This seems a little redundant; you can just call `as_bytes`  on any string in Rust